default/generated/spring-framework/spring-framework-5.x-to-6.0-web-applications.yaml

```diff
@@ -0,0 +1,59 @@
+- ruleID: spring-framework-5.x-to-6.0-web-applications-00001
+  category: potential
+  effort: 1
+  labels:
+  - konveyor.io/source=spring5
+  - konveyor.io/target=spring6+
+  when:
+    or:
+    - java.referenced:
+        pattern: org.springframework.web.bind.annotation.GetMapping
+        location: ANNOTATION
+        annotated:
+          elements:
+            - name: value
+              value: '.*[^\/]$'
+    - java.referenced:
+        pattern: org.springframework.web.bind.annotation.GetMapping
+        location: ANNOTATION
+        annotated:
+          elements:
+            - name: value
+              value: '.*\/$'
```

Comment:
I think this second case is not necessary. With this potential rule what we are trying to do is give a heads up about `GET /some/greeting/` not matching with `@GetMapping("/some/greeting")`, so we need to look for all instances of the later. There would be no need to look for instances of `@GetMapping("/some/greeting/")` because those would only match `GET /some/greeting/` and not `GET /some/greeting`, so there's no ambiguity there.